### PR TITLE
Adds the styled-components module.

### PR DESCRIPTION
**Pull request by jayrbarr**
@timothydadams @JasperC1999 Simple commit to add styled-components functionality.

### CODE REVIEW

File: api.test.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const test = require('jest');
+const app = require('./client/index.jsx');
```
Comment (JasperC1999) [inline api.test.js]: Do you need require statements here? nvm I just noticed it's on the server side.

File: client/components/Review.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+function Review({review}) {
+  return (
```
Comment (JasperC1999) [inline client/components/Review.jsx]: Why a function declaration instead of an arrow function?

File: client/index.jsx [hunk 3]
```diff
@@ -23,7 +25,7 @@ class App extends React.Component {
     }
 
   render() {
-    const listReviews = this.state.reviews.map( review => <div>{review.body}</div>);
+    const listReviews = this.state.reviews.map( (review, index) => <Review key={index.toString()} review={review} />);
```
Comment (JasperC1999) [inline client/index.jsx]: Doesn't eslint bark at you if you use index as a key prop?

File: public/main.js [hunk 4]
```diff
@@ -86,14 +86,27 @@
...
 
 /***/ })
```
Comment (JasperC1999) [inline public/main.js]: are you still committing your main.js?

File: client/index.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+import ReactDOM from 'react-dom';
+import { App } from './components/App';
```
Comment (JasperC1999) [inline client/index.js]: why is destructoring needed here?

File: client/index.js [hunk 6]
```diff
@@ -1,4 +1,4 @@
 import ReactDOM from 'react-dom';
-import { App } from './components/App';
+import App from './components/App.jsx';
```
Comment (JasperC1999) [inline client/index.js]: ah you fixed it here.

File: api.test.js (hunk 1 above)
Comment (jayrbarr) [reply api.test.js]: This is changing a lot right now. Trying to figure out how to set up the testing environment.

File: client/components/Review.jsx (hunk 2 above)
Comment (jayrbarr) [reply client/components/Review.jsx]: yeah, that changed a lot

File: client/index.jsx (hunk 3 above)
Comment (jayrbarr) [reply client/index.jsx]: It does. It gets changed later.

File: public/main.js (hunk 4 above)
Comment (jayrbarr) [reply public/main.js]: Yup. Thanks for helping me fix my gitignore.

File: client/index.js (hunk 6 above)
Comment (jayrbarr) [reply client/index.js]: Yeah, a mistake. I was having trouble getting jest to talk to React and I was trying all sorts of different things from what I was reading. To no avail. I think the testing-library will help a lot. I was trying to take it a step at a time, but maybe that was a mistake in this case.

**Pull request merged**